CreeDictionary/CreeDictionary/templatetags/creedictionary_extras.py

```diff
@@ -52,3 +52,10 @@ def url_for_query_tag(user_query: str) -> str:
         /search?q=w%C3%A2pam%C3%AAw
     """
     return url_for_query(user_query)
+
+
+@register.filter()
+def kbd_text_query_link(text):
```

Comment:
I like this filter!

Could you please a docstring to explain why this exists (i.e., its usage in the FST tool page)?